src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -87,21 +93,69 @@ public void hasModule_moduleInModuleList_returnsTrue() {
         assertTrue(modelManager.hasModule(CS2030));
     }
 
+    @Test
+    public void deleteModule_nullModule_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> modelManager.deleteModule(null));
+    }
+
+    @Test
+    public void deleteModule_moduleNotInModuleList_throwsContactNotFoundException() {
+        // assertThrows(ContactNotFoundException.class, () -> modelManager.deleteModule(null));
+    }
+
+    @Test
+    public void deleteModule_moduleInModuleList_success() {
```

Comment:
Perhaps this test could be removed since its contents are commented out

Reply:
Yes i will take note of them.